### ISSUE TITLE
OBS build failure due to license on Suse 15

### ISSUE THREAD

**Issue by gdha** [cfg2html/cfg2html#161]
```
[   35s] cfg2html.noarch: E: invalid-license (Badness: 100000) GPLv3
[   35s] cfg2html.src: E: invalid-license (Badness: 100000) GPLv3
[   35s] The specified license string is not recognized. Please refer to
[   35s] https://spdx.org/licenses/ for the list of known licenses and their exact
[   35s] spelling.
```
We should use GPL-3.0-or-later instead

**Comment by frankcrawford** [comment]
Pushed update to the RPM spec file in  #162 which should fix it.

**Comment by roseswe** [comment]
@gdha - should be fixed with MR #162 - can you please verify and close this issue? TIA!

**Comment by gdha** [comment]
The RPM build succeeded correctly:
```
[   34s] RPMS/noarch/cfg2html-6.43.2.7.g59171cb-1.1.noarch.rpm
[   34s] SRPMS/cfg2html-6.43.2.7.g59171cb-1.1.src.rpm
[   34s] OTHER/rpmlint.log
[   34s] OTHER/_statistics
[   34s] 
[   34s] lamb09 finished "build cfg2html.spec" at Thu Jan 19 13:49:10 UTC 2023.
```
